client/js/widgets/widget.js

```diff
@@ -30,6 +30,7 @@ export class Widget extends StateManaged {
     this.targetTransform = '';
     this.childArray = [];
     this.propertiesUsedInProperty = {};
+    this.addOverlay = document.getElementById('addOverlay');
```

Comment:
Saving it in `this.addOverlay` seems overkill to me. Technically it's faster though, so it's fine.

In general, I try to use `$('#addOverlay')` in the code instead because it is shorter. Again, technically using `document.getElementById` is faster, so it's fine. Of course, ChatGPT for example doesn't know about those helper functions.